src/tests/test_todo.py

```
@@ -1,10 +1,10 @@
-from src.python.todo import TodoList
+from python.todo import TodoList
 import os
 import unittest
 from datetime import datetime
 from unittest.mock import Mock, patch, call
```

Ruff (0.8.2)

`unittest.mock.call` imported but unused

Remove unused import: `unittest.mock.call`

(F401)